### PR TITLE
Feature/countrycode added geo api

### PR DESCRIPTION
**Pull request by sushil-lifebit**
# Why

we need to country code for backend countries mapping,  so added a country code to get_geocode_info service response

## What 

* added  "countrycode" key to get_geocode_info service
* Addition of unit test

### CODE REVIEW

File: tests/test_get_location.py [hunk 1]
```diff
@@ -203,3 +208,68 @@ def test_get_location_info_returns_empty_list_when_location_found_by_geocoder_ca
     print(response)
     assert mock_get.called
     assert response == expected_output
+
+
+@patch("requests.get")
+def test_get_location_info_returns_right_country_code_when_found_by_geocoder_and_geonames(
```
Comment (ericanand) [inline tests/test_get_location.py:214]: Is this test case just a copy of `test_get_location_info_returns_right_location_when_found_by_geocoder_and_geonames`? Do we need this test case?

**Review by ericanand — COMMENTED** [review]
Some comments - but looking good 👍

File: tests/test_get_location.py (hunk 1 above)
Comment (sushil-lifebit) [reply tests/test_get_location.py:214]: @ericanand  it's kind of duplicate, and my test case will cover by updating another test case,  but still I added it for a different location now it's removed.

**Comment by ericanand** [comment]
Ok this looks good - did you manage to test these changes locally to ensure no errors came up?

**Pull request closed without merging**